google/cloud/storage/_helpers.py

```diff
@@ -147,6 +153,8 @@ def reload(
         self,
         client=None,
         projection="noAcl",
+        if_etag_match=None,
```

Comment:
Question: How strict is the semver? Do I need to add new parameters to the end so as not to break parameter ordering for users who do not specify keys but instead rely on position?

Reply:
Excellent.